### PR TITLE
Calculate some CK routines without using thread private arrays

### PR DESCRIPTION
**Pull request by jrood-nrel**
@esclapez Curious if you could try this in PeleLMeX.

### CODE REVIEW

**Comment by jrood-nrel** [comment]
@PaulMullowney This should give better GPU performance.

**Comment by PaulMullowney** [comment]
> @PaulMullowney This should give better GPU performance.

Hmmm. I wonder if I can backport this into my old PelePhysics branch.

**Comment by jrood-nrel** [comment]
I also inlined CKCVBS and removed thread private arrays, which is another main routine.

**Comment by jrood-nrel** [comment]
These are my results on Crusher with PMF dodecane_lu:
Screen Shot 2023-03-21 at 1 00 36 PM

**Comment by jrood-nrel** [comment]
Actually maybe these don't affect PeleLMeX since it uses a different `reactor_type`?

**Comment by esclapez** [comment]
That's my guess, we might need to do that for the enthalpy routine too.

**Comment by esclapez** [comment]
CKHBMS  and CKCPBS

**Comment by jrood-nrel** [comment]
I can try that. Since I tried to generalize the thermo generation, I was able to do `CKCVBS` easily.

File: Support/ceptr/gen_qss.sh
Comment (malihass) [inline Support/ceptr/gen_qss.sh:1]: Could we remove this file and use the conversion tool from CEPTR instead?

You should be able to regenerate the `qssa.yaml` with 
`poetry run qssa -lq ${PELE_PHYSICS_HOME}/Support/Mechanism/Models/list_qss_mech`

and then regenerate the cpp files with 
`poetry run convert -lq ${PELE_PHYSICS_HOME}/Support/Mechanism/Models/list_qss_mech`

**Comment by jrood-nrel** [comment]
There are some other trivial routines that just do `get_imw` that can be inlined to remove arrays as well. This inlining really increases the code, but it looks beneficial for the GPU.

File: Support/ceptr/gen_qss.sh
Comment (jrood-nrel) [reply Support/ceptr/gen_qss.sh:1]: Sure.

**Comment by jrood-nrel** [comment]
I added CKHBMS and CKCPBS.

**Review by esclapez — APPROVED** [review]
A couple of seconds shaved off in LMeX too, nice !

**Pull request merged**